### PR TITLE
fix(targeting): fix tooltip position in dropdown menu

### PR DESCRIPTION
**Pull request by azuruce**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @azuruce has signed the CLA. Thanks for the pull request!

**Comment by azuruce** [comment]
after fix
image

before fix
image

File: src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -90,6 +90,7 @@ export interface TooltipProps {
 
 type State = {
     isShown: boolean;
+    hasRendered: boolean;
```
Comment (DanDeMicco) [inline src/components/tooltip/Tooltip.tsx]: can you add a comment as to why this is needed for others to understand?

**Comment by ittaishay** [comment]
Approved, but please address @DanDeMicco's comments

File: src/components/tooltip/Tooltip.tsx (hunk 1 above)
Comment (azuruce) [reply src/components/tooltip/Tooltip.tsx]: ouch, just saw this. Will add a comment

**Pull request merged**